### DISCUSSION TITLE
How do I update the URL of starter?

### DISCUSSION THREAD

**Discussion by weensy** [gatsbyjs/gatsby#28692] [Help] [answered]
I can't update the url because docs/starters.yml has been deleted.

**Comment by gillkyle** [comment, marked as answer]
Hey there @vaporwavy 👋  we've been working on an improved ability for users to contribute to the showcases after rearchitecting big portions of the site. That being said, we don't have anything very automated to that end yet.

I see you've got [2 sites in the starter showcase](https://www.gatsbyjs.com/starters?s=vaporwavy&v=2). Could you send the information you'd like to update to my email at kyle.gill@gatsbyjs.com and I'll make sure it gets changed? 🙂

**Reply by weensy** [reply]
Thanks, I sent the email.👍
